README.md

```diff
@@ -1,21 +1,148 @@
-📚 research_LLM 📚
+# Polish benchmark LLM
 
-Project Objective 🎯
-The aim of the "research_LLM" project is to conduct tests and assessments of various Language Models (LLMs - Large Language Models) to understand their performance and applications.
+Polish benchmark LLM is a compact yet powerful platform designed for benchmarking  in polish langue open-source large language models (LLMs)  Equipped with diverse test datasets, it facilitates rigorous and comprehensive evaluation of LLMs. This tool streamlines the process of assessing model performance, enabling researchers and developers to reliably test and share their findings in the dynamic landscape of AI and natural language processing.
+# Table of Contents
 
-Project Stages 📊
+### [Getting Started](#getting-started)
+- [For Liunx](#for-linux-users)
+- [For Windows](#for-Windows-users)
 
-- Model Selection 🧐
-        Selecting different language models for testing.
+### [Run Benchamrk](#run-benchmark)
+- [For Liunx](#for-linux)
+- [For Windows](#for-Windows)
 
- - Data Preparation 📂
-        Gathering and preparing the test dataset.
+### [Below That good first issue](#below-that-good-first-issue)
+- [Prerequisites](#prerequisites)
+- [Installing](#installing)
+- [Running the Tests](#running-the-tests)
+- [Deployment](#deployment)
+- [Built With](#built-with)
+- [Contributing](#contributing)
+- [Versioning](#versioning)
+- [Authors](#authors)
+- [License](#license)
+- [Acknowledgments](#acknowledgments)
 
-- Test Execution 📝
-        Conducting tests on the selected models.
+# Getting Started
+
+This section will guide you through getting a copy of the project up and running on your local machine for development and testing purposes. Before you start, ensure you have Git installed on your system. If not, you can download and install it from [Git's official website](https://git-scm.com/).
+
+## Prerequisites
+
+- Git installed on your machine. 
+- Docker installed for running the application in a container (optional but recommended).
+
+## Setup Instructions
+
+### For Linux Users
+
+Clone the repository to get a local copy on your machine,
+Change into the project directory,
+Run the setup script to configure the environment.
+
+   ```bash
+   git clone https://github.com/knmlprz/Benchmark_LLM.git
+   cd Benchmark_LLM
+   ./trust_me_bro.sh
+   source venv/bin/activate 
+   ``` 
+
+
+### For Windows Users
+
+Clone the repository to get a local copy on your machine,
+Change into the project directory,
+Run the setup script to configure the environment
+
+   ```bash
+git clone https://github.com/knmlprz/Benchmark_LLM.git
+cd Benchmark_LLM
+trust_me_bro.bat
+venv\Scripts\activate.bat
+   ``` 
+
+# Running the Benchmark
+
+### For Linux
+
+**First Time Setup**
+
+Build and start the Docker containers. This step is required only once or after any changes in the Docker configuration.
+```bash 
+sudo docker-compose up --build
```

Comment:
**issue(blocking):** Use `docker compose` instead of `docker-compose`. 

More on this subject here: https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose

Reply:
I don't know about it :heart: